InterGalacticCurrencyConvertor/src/main/java/DataProcessor.java

```diff
@@ -0,0 +1,87 @@
+package main.java;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import main.java.query.Query;
+import main.java.statement.Statement;
+
+/**
+ * Processes user input based on its types: Statement or Query.
+ */
+public class DataProcessor {
+	private static ArrayList<Query> mQueryObjList = new ArrayList<>();
+	private static ArrayList<Statement> mStatementObjList = new ArrayList<>();
+
+	private void loadClasses(String inPackageName, Class<?> inInterfaceName) {
+		try {
+			FindPackageClasses.loadClassesImpInterface(inPackageName,
+			                                           inInterfaceName);
+		} catch (ClassNotFoundException | IOException e) {
+			System.out.println(e);
+			System.exit(-1);
+		}
+	}
+	public static void registerQueryObjects(Query inQuery) {
+		if(inQuery == null) {
```

Comment:
Consider having a simple Preconditions class this this one that checks for null etc.
https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Preconditions.html

Reply:
cannot use external libraries